envbuilder.go

```diff
@@ -156,6 +157,34 @@ func Run(ctx context.Context, options Options) error {
 		}
 	}
 
+	gitURLParsed, err := url.Parse(options.GitURL)
+	if err != nil {
+		return fmt.Errorf("invalid git URL: %w", err)
+	}
+	// If we're cloning over SSH, we need a known_hosts file.
+	if gitURLParsed.Scheme == "ssh" {
+		var knownHostsContent []byte
+		if options.GitSSHKnownHostsBase64 != "" {
```

Comment:
Is it sensitive data? Asking because I see it is encoded on base 64 and I'm wondering why.

Reply:
It's easier to pass data as an environment variable that way; it also follows the existing convention with the docker config JSON.